ui/src/app/shared/components/page.tsx

```diff
@@ -51,9 +51,18 @@ export class Page extends React.Component<{title: string; toolbar?: Toolbar | Ob
 
     private async goToLogin(logout = false) {
         if (logout) {
+            const userInfo = await services.users.get();
```

Comment:
In another comment I've suggested to suppose non OIDC logout in `/auth/logout` API. If suggested is implemented that logout can be just 
```
private async goToLogin(logout = false) {
        if (logout) {
            window.location.href = '/auth/logout';
        } else {
            this.appContext.history.push('/login');
        }
    }
```

Reply:
We would still need to retrieve the constructed URL from the api-server and redirect to the new URL from the front-end right?